rllib/tests/test_supported_spaces.py

```diff
@@ -162,10 +162,6 @@ def test_ppo(self):
         }
         check_support("PPO", config, check_bounds=True, tfe=True)
 
-    def test_pg(self):
```

Comment:
Took this one out. It's already a long, flakey test (due to timeouts). PG doesn't seem important enough to be tested here. PPO, A3C, APPO, Impala should all "cover" for it (similar Policy structure, loss functions, etc..).